src/scripts.js

```diff
@@ -1,32 +1,62 @@
 import './styles.css';
-import apiCalls from './apiCalls';
+import { fetchAll } from './apiCalls';
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png';
-import Recipe from './classes/Recipe';
 import RecipeRepository from './classes/RecipeRepository';
-import recipeData from './data/recipes';
 import User from './classes/User';
-import usersData from './data/users';
-
-// ------------------- GLOBAL VARIABLES
-const recipeRepo = RecipeRepository.fromRecipeData(recipeData);
-const user = changeUser(usersData)
-let tag;
-let tagList = [];
 
 // --------------------QUERY SELECTORS
 const modal = document.querySelector('#modal');
 const close = document.querySelector('#close');
 const recipeSection = document.querySelector('#recipeSection');
 const tagsContainer = document.querySelector('#tagsContainer');
-const searchAllRecipesButton = document.querySelector('#searchAllRecipesButton');
+const searchAllRecipesButton = document.querySelector(
+  '#searchAllRecipesButton'
+);
 const searchCookbookButton = document.querySelector('#searchCookbookButton');
 const allRecipesSearchBar = document.querySelector('#allRecipeSearch');
 const cookbookSearchBar = document.querySelector('#cookbookSearch');
-const allRecipesTab = document.getElementById('tabAllRecipes')
-const cookbookTab = document.getElementById('tabCookbook')
+const allRecipesTab = document.getElementById('tabAllRecipes');
+const cookbookTab = document.getElementById('tabCookbook');
+
+// ------------------- GLOBAL VARIABLES
+const store = {
+  userData: [],
+  ingredientsData: [],
+  recipeData: [],
+  recipeRepo: new RecipeRepository(),
+  user: new User(),
+  tagList: [],
+  tag: '',
+};
+
+// --------------------------------------------- Initialize App
+const initializeApp = () => {
```

Comment:
this makes so much more sense to have this in one function that happens on page load.